content/blog/2021/diversity.md

```diff
@@ -0,0 +1,57 @@
+Title: Diversity track schedule at PyCon India 2021
+Date: 2021-09-07 17:00
+Category: 2021
+Slug: diversity-and-inclusion
+Summary: The schedule is here for the Diversity track for this year's premier Python conference, **PyCon India 2021**.
+
+[PyCon India](https://in.pycon.org/2021/) is a grass-roots, volunteer-driven conference for the Python programming language. Python communities across the nation have come together to help plan, organize, manage, and run the online conference. This year, we plan to make this mega-event even more diverse and inclusive by giving folks in the community a stage to share their ideas and insights.  Henceforth, for the second year running, PyCon India will continue to have a Diversity track. You can glance through the final schedule for all D&I sessions along with **session details** below:
+
+## Day 1 (18th Sept):
+
+### Title: How to ace problem-solving – perspective of a non-CS engineer
+
+**Time**: 14:00 to 14:30
+**Theme**: Women in Competitive Coding
+**Speaker**: Anushi Maheshwari
+**Session Details** - https://in.pycon.org/cfp/2021/proposals/how-to-ace-problem-solving-perspective-of-a-non-cs-engineer~dyPX7/
+
+### Title: Neurodiversity Demystified
+
+**Time**: 15:30 to 16:30
+**Theme**: Neurodiversity Awareness Building
+**Speaker**: Joel S Godi
+**Session Details** - https://in.pycon.org/cfp/2021/proposals/neurodiversity-demystified~dwprR/
+
+### Title: PhD - and the many possibilities ahead
+
+**Time**: 18:30 to 19:00
+**Theme**: Women in Research
+**Speaker**: Ananya Mandal
+**Session Details** - https://in.pycon.org/cfp/2021/proposals/phd-and-the-many-possibilities-ahead~axk19/
+
+---
+
+## Day 2 (19th Sept)
+
+### Title: Women in tech careers - taking that extra leap
+
+**Time**: 14:00 to 14:30
+**Theme**: Women in technology
+**Speaker**: Manaswini Das
+**Session Details** - https://in.pycon.org/cfp/2021/proposals/women-in-tech-careers-taking-that-extra-leap~azpXY/
+
+### Panel: Neurodiversity & Inclusion
+
+**Time**: 15:30 to 16:30
+**Theme**: Inclusion Strategies & Best Practices
+**Moderator and Panelists**: Panchali Banerjee, Joel S Godi, Mark Charlesworth, Asha Sreedhar
+**Panel Details** - https://in.pycon.org/cfp/2021/proposals/panel-neurodiversity-inclusion~av2Zm/
+
+### Panel: Diversity in Cybersecurity
+
+**Time**: 16:30 to 17:30
+**Theme**: The importance of Diversity and Inclusion in Tech
+**Moderator and Panelists**: Vandana Verma, Shira Shamban, Hitesh Dharmdasani, Judy Ngure
+**Panel Details** - https://in.pycon.org/cfp/2021/proposals/panel-diversity-in-cybersecurity~erk9E/
+
```

Comment:
Can we put it under markdown table and just hyperlink the session details?

Reply:
@raukadah Sure yeah, that sounds like a better idea.